Add an introductory paragraph (#779)
kyma-project/community: concepts/modularization/modularization_overview.md

title: Modularization in Kyma
---

## Overview

> The out-of-the-box Kyma offered a fixed set of mandatory preconfigured components whose development rhythm was synchronized and determined by the release schedule. With the modular approach, Kyma components become modules, each providing one functionality developed independently of the other ones. This approach also boosts Kyma’s extensibility and scalability. You can install and configure exactly the modules you want, making the installation very lightweight and adjusted to your business needs.  

Reviewer: ```suggestion
Classic Kyma offered a fixed set of mandatory preconfigured components with a synchronized development and release schedule. With the modular approach, Kyma components become modules, each providing one functionality developed independently of the other ones. The modular approach also boosts Kyma’s extensibility and scalability. You can install and configure exactly the modules you want, making the installation very lightweight and adjusted to your business needs.  
```

I think the first sentence could be simplified slightly.
"Classic" is a word I'd suggest, because "out-of-the-box" doesn't imply that that's the "older" approach which is going to be replaced by modular Kyma at some time.
